integrations-and-supported-tools/great_expectations/notebooks/Neptune_Great_Expectations.ipynb

```diff
@@ -0,0 +1,548 @@
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "context = gx.get_context()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Upload Context configuration to Neptune"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "run[\"gx/context/config\"] = context.get_config().to_json_dict()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The above code cell logs the GX data context configuration to the `gx/context/config` namespace in the Neptune run."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Connect to data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "validator = context.sources.pandas_default.read_csv(\n",
+    "    \"https://raw.githubusercontent.com/great-expectations/gx_tutorials/main/data/yellow_tripdata_sample_2019-01.csv\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create Expectations"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "validator.expect_column_values_to_not_be_null(\"pickup_datetime\")\n",
+    "validator.expect_column_values_to_be_between(\"passenger_count\", min_value=1, max_value=6)\n",
+    "validator.save_expectation_suite(\n",
+    "    discard_failed_expectations=False,\n",
+    "    discard_catch_exceptions_kwargs=False,\n",
+    "    discard_include_config_kwargs=False,\n",
+    "    discard_result_format_kwargs=False,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Log Expectations to Neptune"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "expectation_suite = validator.get_expectation_suite().to_json_dict()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "run[\"gx/meta\"] = expectation_suite[\"meta\"]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "run[\"gx/expectations/expectations_suite_name\"] = expectation_suite[\"expectation_suite_name\"]\n",
+    "\n",
+    "for idx, expectation in enumerate(expectation_suite[\"expectations\"]):\n",
+    "    run[\"gx/expectations\"][idx] = expectation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The above code cell logs the GX Expectations suite name to the context configuration to the `gx/expectations/expectations_suite_name` field of the run, and creates a numbered folder for each expectation in the `gx/expectations` namespace."
```

Comment:
Some typo in the structure? (I see two consecutive `to the`s)

Reply:
ah.. thanks for noticing!